src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -2199,16 +2304,6 @@ IsSelectClause(Node *clause)
 	Var *firstColumn = NULL;
 	Index firstColumnTableId = 0;
 	bool isSelectClause = true;
-	NodeTag nodeTag = nodeTag(clause);
```

Comment:
If we are sure that no sublinks will be passed into this function, I think we can remove this check.